changelog.txt

```diff
@@ -17,6 +17,7 @@
 * Fix - Update Ukraine state mapping list
 * Fix - Use same default locations for Amazon Pay express checkout
 * Dev - Add configuration and workflow for PHPStan
+* Fix - Calculate subtotal correctly in express checkout when itemized display is disabled, preventing Google Pay from showing incorrect totals
```

Comment:
I could reproduce the issue with Apple Pay, Link, and Amazon Pay, so I think we need to make the test more general.
```suggestion
* Fix - Calculate subtotal correctly in express checkout
```